server/apps/main/urls.py

```diff
@@ -14,7 +14,7 @@
         name='view_experiences'),
     url(r'^moderate_public_experiences/?$', views.moderate_public_experiences,
         name='moderate_public_experiences'),
-    url(r'^upload/?$', views.upload, name='upload'),
```

Comment:
As a head's up, this change breaks the currently existing `/main/list` view. But that view might be removed in any case in favor of this PR if I understand it correctly.

Reply:
@gedankenstuecke correct. There was some redundancy in the code. There were some pages that never actually were loaded. (`list.html` included). We adapted some code from the list view/html pages for the new `my_stories`.

It seems that there were some efforts to abstract the front-end into a central 'application' (`application.html`), which was to be extended in sub-pages (e.g. for `list.html`). I'm not sure about the chronology here, because they weren't actually used. 

I was going to keen the redundant pages/views for now and tidy things up when we are closer to an MVP since we found code in the unused pages to be useful.

Reply:
Got it, just wanted to flag it in case it's a real issue. But then we can ignore this for now and leave the clean up of unused things for closer to the MVP